pipeline/common_stages.groovy

```diff
@@ -463,8 +467,47 @@ def deploy_mig_controller_on_both(
   }
 }
 
+/*
+  Builds operator images, creates Quay
+  application, pushes application
 
-def execute_migration(e2e_tests, source_kubeconfig, target_kubeconfig) {
+  directory [string] : Directory of operator repo to build
+*/
```

Comment:
build custom operator image in following function. 

use env variables `MIG_OPERATOR_REPO` and `MIG_OPERATOR_BRANCH` 

each build increments the Quay application version number by 1